src/icesat2_tracks/local_modules/m_general_ph3.py

```diff
@@ -1568,33 +1568,33 @@ def gen_log_space(limit, n):
 
 def linefit2Points(time_lin, f, data, f1, f2, f_delta=None,  plot=False):
     if isinstance(time_lin[0], np.datetime64):
-        print('type is numpy.datetime64', time_lin.shape)
+        _logger.debug('type is numpy.datetime64', time_lin.shape)
         time_lin=time_lin.astype('m8[s]').astype(int)
-        #print(ttt)
+        #_logger.debug(ttt)
 
     #if isinstance(time_delta, np.timedelta64):
     #        time_delta=time_delta.astype('m8[s]').astype(int)
-    #        #print(time_delta)
+    #        #_logger.debug(time_delta)
 
 
     if f.shape [0] != data.shape [0]:
-        print('ERROR: shapes are not correct')
-        print(f.shape, time_lin.shape, data.shape )
+        _logger.debug('ERROR: shapes are not correct')
```

Comment:
Use a higher logging level here? Though an error is not thrown afterwards idk.
```suggestion
        _logger.error('ERROR: shapes are not correct')
```

Reply:
Done